TypeCobol/Compiler/SqlCodeElements/SqlCodeElementType.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.CodeElements;
+
+namespace TypeCobol.Compiler.SqlCodeElements
+{
+    /// <summary>
+    /// The Sql syntax elementary code elements
+    /// </summary>
+    public enum SqlCodeElementType
+    {
+        FirstSqlCodeElementType = CodeElementType.CommitStatement,
```

Comment:
Remove this. See my previous remark on enum design.
Also this is strange, i see that this works as an extension to CodeElementType enum but why do we need a second enum ? To find whether a statement is an SQL statement or not, we could test the type against `SqlStatementElement`. To test the type of a statement without using a cast we use the `CodeElementType` enum (which contains CommitStatement).

Reply:
removed in 5a26682